lib/commands/help-search.js

```diff
@@ -28,7 +28,7 @@ class HelpSearch extends BaseCommand {
 
   async exec (args) {
     if (!args.length)
-      return this.npm.output(this.usage)
```

Comment:
This one was correct imo.

```sh
$ npm help-search
npm help-search

Search npm help documentation

Usage:
npm help-search <text>

Options:
[-l|--long]

Run "npm help help-search" for more info
```

Reply:
Actually no I'm now on the other side of the fence, this is the correct update.  We should NOW be able to make `lib/utils/error-message.js` standardize how we output usage errors.